projects/easywsclient/project.yaml

```diff
@@ -0,0 +1,9 @@
+homepage: "https://github.com/dhbaird/easywsclient"
+primary_contact: "dhbaird@gmail.com"
```

Comment:
do you have an approval from this person? if they have a GitHub account, it's a good practice to mention them on the PR to let them take a look and be aware of the integration